.gitignore

```diff
@@ -11,6 +11,7 @@ test/**/test_output
 
 node_modules/
 npm-debug.log
+.yarn-lock
```

Comment:
`yarn.lock` should be gitignored; but I'm not sure what `.yarn-lock` is nor why it would occur in a project not using yarn?